java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1409,9 +1409,43 @@ public ColumnVector asDate64() {
    * @return A new vector allocated on the GPU
    */
   public ColumnVector asTimestamp(TimeUnit unit) {
+    if (type == DType.STRING)
+      return asTimestamp(unit, "%Y-%m-%dT%H:%M:%SZ");
```

Comment:
Could you please at least document the default timestamp format you are using in the javadocs.  What happens if the format does not match what is provided?